test/wasmBenchmarker/ctests/kNucleotide.c

```diff
@@ -316,4 +315,88 @@ const char *input_3 =
         "tctgtcccagaacagctccacaagtttttttacagccgaaacccctgtgtgaatcttaat"
         "atccaagcgcgttatctgattagagtttacaactcagtattttatcagtacgttttgttt"
         "ccaacattacccggtatgacaaaatgacgccacgtgtcgaataatggtctgaccaatgta"
+        "ggaagtgaaaagataaatat"
```

Comment:
Is this a random addition? This code is a human dna :)

Reply:
Yes, this is random. I'll look after another human DNA-s